.github/workflows/check.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: actions/setup-java@v2
+      - uses: actions/setup-java@v3
         with:
           distribution: temurin
           java-version: 19
```

Comment:
Any reason we shouldn't be using java 17?

Reply:
Well java 19 is backwards compatible with java 17 so there *shouldn't* be any issues, I'm using java 19 on my own machines so if there's ever any issues I'll probably fix them

Reply:
It'd be best to stay with Temurin-17 so we're more correctly simulating what runs on the robot. Last year we ran into JDK-specific bugs, and I'd want to avoid missing any of those this year.